### PR TITLE
added workflow, delete dist folder

### PR DESCRIPTION
**Pull request by shafeeqd959**

### CODE REVIEW

**Comment by netrajpatel** [comment]
@shafeeqd959 @cs-raj Is this still required or we have taken care of these changes in any other PR?

**Comment by shafeeqd959** [comment]
@netrajpatel , this is valid, we can take the changes

**Comment by shafeeqd959** [comment]
@netrajpatel , we need to add the npm token in this repo to make this release workflow work

**Comment by aman19K** [comment]
@cs-raj Can you pls run npm audit fix ? I saw some SRE issues last time which could be resolved just by audit fix.

**Comment by cs-raj** [comment]
> @cs-raj Can you pls run npm audit fix ? I saw some SRE issues last time which could be resolved just by audit fix.



> @cs-raj Can you pls run npm audit fix ? I saw some SRE issues last time which could be resolved just by audit fix.

@aman19K done

**Comment by abhinav-from-contentstack** [comment]
Bumping version as `dist` folder is deleted

**Comment by abhinav-from-contentstack** [comment]
Reverting commit, version was already bumped

**Review by abhinav-from-contentstack — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "debug": "^4.3.4",
     "lodash": "^4.17.15",
-    "mongodb": "^4.9.0"
+    "mongodb": "^5.7.0"
```
Comment (antonyagustine) [inline package.json:10]: @shafeeqd959 / @cs-raj  Have we conducted validation for the breaking changes introduced in versions v5 and above?

https://github.com/mongodb/node-mongodb-native/releases/tag/v5.0.0
https://github.com/mongodb/node-mongodb-native/compare/v4.9.0...v5.7.0

cc @netrajpatel

Comment (cs-raj) [reply package.json:10]: @antonyagustine  I tested it. it worked fine
cc: @shafeeqd959 @netrajpatel

**Review by aravindbuilt — APPROVED** [review]

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**